src/Products/Viewer/Util/DbInputHandler.cs

```diff
@@ -0,0 +1,109 @@
+﻿using GroupDocs.Viewer.MVC.Products.Common.Config;
+using GroupDocs.Viewer.MVC.Products.Common.Entity.Web;
+using GroupDocs.Viewer.MVC.Products.Common.Resources;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.IO;
+
+namespace GroupDocs.Viewer.MVC.Products.Viewer.Util
+{
+    public class DbInputHandler : IInputHandler
+    {
+        private readonly GlobalConfiguration globalConfiguration;
+
+        public DbInputHandler(GlobalConfiguration globalConfiguration)
+        {
+            this.globalConfiguration = globalConfiguration;
+        }
+
+        public string GetFileName(string guid)
+        {
+            string ext;
+            using (SqlConnection cn = new SqlConnection(@"Data Source =.\SQLEXPRESS; Initial Catalog = tempdb; Integrated Security = True"))
+            using (SqlCommand cm = cn.CreateCommand())
+            {
+                cm.CommandText = @"
+                    SELECT Ext
+                    FROM   tempdb.dbo.Files
+                    WHERE  Id = @Id";
+                cm.Parameters.AddWithValue("@Id", guid);
+                cn.Open();
+                ext = (cm.ExecuteScalar() as string).TrimEnd();
+            }
+
+            return guid + "." + ext;
+        }
+
+        public List<FileDescriptionEntity> GetFilesList()
+        {
+            var filesList = new List<FileDescriptionEntity>();
+
+            SqlConnection conn = new SqlConnection(@"Data Source =.\SQLEXPRESS; Initial Catalog = tempdb; Integrated Security = True");
+            conn.Open();
+
+            SqlCommand command = new SqlCommand("select Id, Ext from tempdb.dbo.Files", conn);
+            // int result = command.ExecuteNonQuery();
+            using (SqlDataReader reader = command.ExecuteReader())
+            {
+                while (reader.Read())
+                {
+                    Console.WriteLine(String.Format("{0}", reader["Id"]));
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=6117781)